tests/devices/test_default_mixed.py

```diff
@@ -911,6 +911,7 @@ def test_identity_skipped(self, mocker):
             qml.probs(op=qml.Y(0)),
             qml.probs(op=qml.X(0) @ qml.Y(1)),
             qml.vn_entropy(wires=[0]),
+            qml.vn_entanglement_entropy(wires0=[1], wires1=[0]),
```

Comment:
Can we also test multi-wire subsystems?

Reply:
I'll add it, but it doesn't seem relevant to the purpose of the test.

Reply:
Guess not, it's fine if you leave this as is then.